termine/announce.go

```diff
@@ -147,25 +116,20 @@ func sendAnnouncement(subject string, msg []byte) {
 		log.Println("Output von Sendmail:")
 		io.Copy(os.Stderr, stdout)
```

Comment:
If we are handing errors upwards anyways, it would be consequenial to return some error here as well, wouldn't it? Can be implemented later though, but we should file an issue then, so that it is done eventually.